h2o-r/tests/testdir_algos/automl/runit_automl_args.R

```diff
@@ -25,7 +25,7 @@ automl.args.test <- function() {
   print("Check arguments to H2OAutoML class")
 
   #print("Try without a y")
-  #expect_failure(h2o.automl(training_frame = train,
+  #expect_failure(h2o.automl(training_rframe = train,
   #                          max_models = max_models,
   #                          project_name = "aml0"))
 
```

Comment:
Is the above commented code needed?

Reply:
probably not, I'm usually in favour of deleting unused and/or commented code, but I saw too much of it :)

Reply:
Ah, I see. I would preferably remove if not needed. Plus its in version control if anyone ever needed it again. :)